modules/gapi/src/backends/onnx/gonnxbackend.cpp

```diff
@@ -491,7 +499,12 @@ ONNXCompiled::ONNXCompiled(const gapi::onnx::detail::ParamDesc &pp)
     // Create and initialize the ONNX session
     Ort::SessionOptions session_options;
     this_env = Ort::Env(ORT_LOGGING_LEVEL_WARNING, "");
+#ifndef _WIN32
     this_session = Ort::Session(this_env, params.model_path.data(), session_options);
+#else
+    std::wstring w_model_path(params.model_path.begin(), params.model_path.end());
+    this_session = Ort::Session(this_env, w_model_path.data(), session_options);
+#endif
```

Comment:
@dmatveev, It is a fix for successful build on Windows.

Reply:
ort::Session on Windows expects wchar_t* instead char*

Reply:
Isn't it more complex than that?

Internets say something about
```c++
std::wstring_convert<std::codecvt_utf8<wchar_t>>().from_bytes(input_str);
```

cc: @alalek

Reply:
Currently I don't have BKMs for that.
Try to validate that on your side (original "fix" doesn't look right in case of UTF-8 `std::string`).

---

BTW, modern Windows versions may support UTF-8 encoding from the box: https://www.reddit.com/r/cpp/comments/g7tkd7/state_of_utf8_support_on_windows/

Reply:
@TolyaTalamanov can you please point to `FText{}` implementation here?

Reply:
Have a look on this: https://github.com/opencv/opencv/blob/master/modules/gapi/test/render/gapi_render_tests_ocv.cpp#L100

Reply:
Fixed. This looks like implementation in `gapi_render_tests_ocv.cpp` now.